Refusing to install package with name "react-facebook-login" under a package also called "react-facebook-login"

Following the instructions on the [homepage](https://www.npmjs.com/package/react-facebook-login):

```
git clone https://github.com/keppelen/react-facebook-login.git && cd react-facebook-login
npm install react react-dom react-facebook-login --save
```
with node 7.2.0 and npm 4.0.5 yields:

```
~/react-facebook-login$ npm install react react-dom react-facebook-login --save
npm ERR! Linux 3.13.0-85-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "react" "react-dom" "react-facebook-login" "--save"
npm ERR! node v7.2.0
npm ERR! npm  v4.0.5
npm ERR! code ENOSELF

npm ERR! Refusing to install package with name "react-facebook-login" under a package
npm ERR! also called "react-facebook-login". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR! 
npm ERR! For more information, see:
npm ERR!     <https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm>
npm ERR! 
```

--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 ## Getting Started
 ```shell
 git clone https://github.com/keppelen/react-facebook-login.git && cd react-facebook-login
-npm install react react-dom react-facebook-login --save
+npm install react react-dom react-facebook-login --save --force
 ```
 
 ## Development

